library/cc/bridge_utility.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "headers.h"
+#include "library/common/types/c_types.h"
+
+namespace Envoy {
+namespace Platform {
+
+envoy_data buffer_as_envoy_data(const std::vector<uint8_t>& data);
```

Comment:
My inclination is not to introduce std::vector<uint8_t> as a buffer representation unless there's a good pythonic reason for doing so. Can we simply directly use envoy_data?

Reply:
yeah--this translates into `List[int]` in Python, but Eventually™ you can tell pybind11 to reinterpret a `std::vector<uint8_t>` as a `bytes` object.

a better approach (if it's supported by pybind11) is to just provide a `bytes` view onto an `envoy_data` instance

Reply:
Okay, cool - I'd just prefer not to introduce yet another buffer representation if at all possible.